src/front/glsl/ast.rs

```diff
@@ -123,9 +123,13 @@ pub enum HirExprKind {
         tgt: Handle<HirExpr>,
         value: Handle<HirExpr>,
     },
-    IncDec {
-        increment: bool,
+    /// A prefix/postfix operator like `++`
+    Infix {
```

Comment:
This is the exact opposite of infix!

Infix is the + in 2+2, postfix is 2+, prefix is +2

Reply:
hmm, you're right english isn't my first language so I confused the meaning, doesn't seem like english has a direct equivalent of the word I initially thought of.